tests/python/contrib/test_clml/infrastructure.py

```diff
@@ -155,9 +173,9 @@ def build_and_run(
     for _ in range(no_runs):
         gen_module.run()
         out.append([gen_module.get_output(i) for i in range(outputs)])
-    time_f = gen_module.module.time_evaluator("run", device.device.cl(0), number=1)
-    cost = time_f().mean
-    print("%g secs/iteration\n" % cost)
+    # time_f = gen_module.module.time_evaluator("run", device.device.cl(0), number=1)
+    # cost = time_f().mean
+    # print("%g secs/iteration\n" % cost)
```

Comment:
Why did you remove it?

Reply:
This affects CI time and also to get ideal numbers we need to set ```number``` to a higher value too. 
For now, I am leaving it for developers to uncomment and use.